SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -136,3 +204,22 @@ TArray<SpatialLoadBalanceEnforcer::AclWriteAuthorityRequest> SpatialLoadBalanceE
 
 	return PendingRequests;
 }
+
+void SpatialLoadBalanceEnforcer::QueueAclAssignmentRequest(const Worker_EntityId EntityId)
+{
+	UE_LOG(LogSpatialLoadBalanceEnforcer, Verbose, TEXT("Queueing ACL assignment request for entity %lld on worker %s."), EntityId, *WorkerId);
+	AclWriteAuthAssignmentRequests.Add(WriteAuthAssignmentRequest(EntityId));
+}
+
+bool SpatialLoadBalanceEnforcer::AclAssignmentRequestIsQueued(const Worker_EntityId EntityId)
+{
+	return AclWriteAuthAssignmentRequests.ContainsByPredicate([EntityId](const WriteAuthAssignmentRequest& Request) { return Request.EntityId == EntityId; });
```

Comment:
nit - the usage of this array suggests it might be better as an array?

Reply:
I think there's an assumption that this array will generally be small. This is also how it was used before, but I could see this making more sense as a set as well. Actually I like the sound of a set will change it to that

Reply:
a map makes way more sense

Reply:
I've swung back round to a set

Reply:
@samiwh loves sets

Reply:
went with an array because the type was too ambiguous for linux to hash, even though it built fine on windows 😞